src/mmds/src/token_headers.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use micro_http::{HttpHeaderError, RequestError};
+use std::collections::HashMap;
+use std::fmt;
+use std::fmt::{Display, Formatter};
+use std::result::Result;
+
+const X_METADATA_TOKEN: &str = "X-metadata-token";
+const X_METADATA_TOKEN_TTL_SECONDS: &str = "X-metadata-token-ttl-seconds";
```

Comment:
1. Can we define these consts in the `TokenHeaders` implementation as suggested in the previous comment ?
2. Do we still need `TokenHeader` if we define these consts ?

Reply:
Seems like I misunderstood your initial comment. So you are suggesting to use constants instead of enums for the headers variants, right?
I don't have a strong preference to keep the initial implementation and it might make the code easier to understand if ditch the `TokenHeader` struct. I made the changes.

Reply:
Yes, that's what I meant. Looks good.